tests/plugins/pastefromlibreoffice/manual/main.md

```diff
@@ -1,4 +1,4 @@
-@bender-tags: feature, 4.14.0, libreoffice
```

Comment:
It would be easier to verify if content was pasted correctly if the editor window was bigger. Let's set editor's height to `500` maybe?

Reply:
We are keeping the default height for every pasting manual test. I'm for keeping it for consistency since you can just use a resize handler.

Reply:
Ok, using resizer makes sense. We just need to enable it in `tests/plugins/pastefromgdocs/manual/pastefromgdocs` then.